### DISCUSSION TITLE
Allow users to undo changes in the browsing history with CTRL or CMD Z, like they can with bookmarks

### DISCUSSION THREAD

**Discussion by MoltenMonster** [Floorp-Projects/Floorp#424] [Feature Request]
Currently in Floorp, when a user changes, renames, and or deletes bookmarks, they can press the undo or redo keys (CTRL Z and CTRL Y for instance) in the bookmarks menu (under Library > Bookmarks > Manage Bookmarks) to undo and redo changes made to bookmarks during the current browsing section. Floorp should also have the capability in the history menu when users make changes to history entries.

**Comment by menndouyukkuri** [comment]
I don't think browsing history is something that you can flexibly edit.
